templates/spin_sidebar_v3.hbs

```diff
@@ -111,6 +111,26 @@
                                         {{/if}} href="{{site.info.base_url}}/spin/v3/redis-trigger">The Redis
                                         Trigger</a>
                         </li>
+                        <li><a {{#if (active_project request.spin-full-url "/spin/v3/command-trigger" )}} class="active"
+                                        {{/if}} href="{{site.info.base_url}}/spin/v3/command-trigger">The Command
```

Comment:
What do you reckon is a good way to order the TOC here?  The current scheme seems to be:

* built ins, in order of importance
* Fermyon-authored, in alphabetical? order
* non-Fermyon-authored, unordered

This kind of requires users to know the provenance of their trigger in order to find it, which feels awkward.

On the other hand, strict alphabetical would make HTTP disappear into the middle of the list, which doesn't feel like it well serves the most common user needs.

Reply:
Maybe we can simple add "Available triggers" to the sidebar, and link to that page from each trigger instead of listing all there... feels smarter

Reply:
Yeah I like that, it that would also give us a nice landing page for inbound links (and a place to explain the different statuses of the different triggers)